packages/grid/src/vaadin-grid-mixin.js

```diff
@@ -270,6 +270,33 @@ export const GridMixin = (superClass) =>
         reorderElements: true,
       });
 
+      // Chromium based browsers cannot properly generate drag images for elements
+      // that have children with massive heights. This workaround prevents crashes
+      // and performance issues by excluding the items from the drag image.
+      // https://github.com/vaadin/web-components/issues/7985
+      document.addEventListener(
+        'dragstart',
+        (e) => {
+          // The dragged element can be the element itself or a parent of the element
+          if (e.target !== this && !e.target.contains(this)) {
+            return;
+          }
+          // The threshold value 20000 provides a buffer to both
+          //   - avoid the crash and the performance issues
+          //   - unnecessarily avoid excluding items from the drag image
+          if (this.$.items.offsetHeight > 20000) {
+            const initialMaxHeight = this.$.items.style.maxHeight;
+            // Momentarily hides the items until the browser starts generating the
+            // drag image.
+            this.$.items.style.maxHeight = '0';
+            requestAnimationFrame(() => {
+              this.$.items.style.maxHeight = initialMaxHeight;
+            });
+          }
+        },
+        { capture: true },
```

Comment:
Is listening on capture phase necessary?

Reply:
It seems that otherwise it will does reach the listeners.